accounts-db/src/accounts_db.rs

```diff
@@ -4650,18 +4648,18 @@ impl AccountsDb {
                 shrink_slots.insert(usage.slot, Arc::clone(store));
             }
         }
+
         measure.stop();
-        inc_new_counter_debug!(
-            "shrink_select_top_sparse_storage_entries-ms",
-            measure.as_ms() as usize
-        );
-        inc_new_counter_debug!(
-            "shrink_select_top_sparse_storage_entries-seeds",
-            candidates_count
-        );
-        inc_new_counter_debug!(
-            "shrink_total_preliminary_candidate_stores",
-            total_candidate_stores
+        datapoint_info!(
+            "shrink_select",
+            ("select_time_us", measure.as_us(), i64),
+            ("candidates_count", candidates_count, i64),
+            ("selected_count", shrink_slots.len(), i64),
+            (
+                "deferred_to_next_round_count",
+                shrink_slots_next_batch.len(),
+                i64
+            )
```

Comment:
Maybe not. 
FWIK. ShrinkStats is more like a aggregated view from accounts-db perspective, which aggregates all the calls to shrink and report a sum of the timing and other stats for all the shrink invocations per a period. While this datapoint is to report timing and stats from just *one* invocation of shrink, such as one shrink call from ABS. Thus, it can be thought of as a more detailed stats for `handle_snapshot_requests-timing.shrink_time`.